Articles/Blog/RseStoriesPodcast.md

```diff
@@ -10,75 +10,73 @@
 
 ### We look to experts for knowledge, but do they know the whole picture?
 
-I woke up one morning in September 2019 with something on my mind. I have read several
-papers and heard talks and other "academically approved" presentation formats that purport to define what
-a research software engineer (RSE) is. But the definitions don't reflect my experience.
-Nor do they reflect the experiences of many of my peers. Instead, they present a  narrow
-understanding of a role that is immensely rich and changing. I have seen RSEs working at national and academic labs, involved in everything from documentation and user support to high-performance computing and
-open source development. 
-What I see in the real world
-doesn't reflect what the experts are advocating — research software engineers do not
+I woke up one morning in September of 2019 with something on my mind. I had seen several
+papers, talks, and other "academically approved" presentation formats that were defining what
+a research software engineer (RSE) is. What was the problem? It didn't reflect my experience.
+It didn't reflect the experiences of many of my peers, and presented a very narrow
+understanding of a role that was immensely rich and changing. Could it be that the community
+would need a source different from publication to derive our truth? What exactly is a
+research software engineer anyway? Did I know? I had seen RSEs in high performance computing,
+open source development, and of course at national and academic labs. These folks
+worked on everything from documentation to user support. What I saw in the real world
+didn't reflect what the experts were advocating - research software engineers did not
 always exist on some dimension between software engineer and researcher.
 
 ### If there is missing information, where can we find it?
 
-This of course isn't a new problem.  I have been chewing on it for a few months now.
```

Comment:
Again - this needs to be past tense. I am reflecting back on a memory from 2019.